infrastructure/local/docker-compose.yml

```diff
@@ -48,9 +45,10 @@ services:
       ILP_ADDRESS: test.rafiki
       STREAM_SECRET: BjPXtnd00G2mRQwP/8ZpwyZASOch5sUXT5o0iR5b5wU=
       ADMIN_KEY: admin
-      PUBLIC_HOST: http://backend
-      OPEN_PAYMENTS_URL: op.rafiki.test
-      WEBHOOK_URL: http://mock-account-provider/webhooks
+      ADMIN_HOST: rafiki.test:3001
+      OPEN_PAYMENTS_HOST: wallet.rafiki.test:3003
+      CONNECTOR_HOST: ilp.rafiki.test:3002
+      WEBHOOK_URL: https://end3sf6r22xva.x.pipedream.net
```

Comment:
Did `WEBHOOK_URL` need to change?

Reply:
We didn't talk about `WEBHOOK_URL` but as far as I understand we don't need to change it.